.github/workflows/ci.yml

```diff
@@ -0,0 +1,56 @@
+name: CI
+on:
+  # Run on all pushes to master (including merge of PR)
+  push:
+    branches: master
+  # Run on all changes in pull requests
+  pull_request:
+  # Run CI once per day (at 06:00 UTC)
+  # schedule:
+  #   - cron: '0 6 * * *'
+
+jobs:
+  nikola:
+    name: Basic tests (Python ${{ matrix.python }} on ${{ matrix.image }})
+    strategy:
+      matrix:
+        python:
+          - 3.5
+          - 3.6
+          - 3.7
+          - 3.8
+        image:
+          - unbutu-18.04
```

Comment:
1. There’s a typo. (u**nb**utu)
2. We should limit the Ubuntu versions we use as much as possible. It looks like you can install everything in 20.04: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md — so we should use that, or `ubuntu-latest` (which will become 20.04 soon: https://github.com/actions/virtual-environments/issues/1816)
3. This should use `setup-python`, as in this example: https://docs.github.com/en/free-pro-team@latest/actions/guides/building-and-testing-python

I’ll try to commit a fix to all those issues.

Reply:
Please don't, I'm actively working on this.

Reply:
This issue was assigned to me. In the future, please ask before working on an assigned issue.

Reply:
I used the same Ubuntu versions that were mentioned in the travis config. I was using this list: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on

Reply:
Travis uses 16.04 for 3.5 because they don’t offer 3.5 on newer versions. The provider change lets us switch to a more modern OS version for all tests.